### PR TITLE
Portuguese translations

### LINKED ISSUE #329: Complete and update Portuguese translation (10/2020)
Review, complete and update existing Portuguese translations of the app. :portugal: 

The _Translation Editor_ in _Android Studio_ let's you easily identify the **missing** translations as shown in the screenshot. Existing translations might be **outdated** or **incomplete** and must be checked critically.

Android-Studio-Translation-Editor

Please use the automatic formatting for the string files which Android Studio provides.

- [ ] Review, complete and update existing translations in `app/src/main/res/values-pt` (ignore `placeholder` strings)
  - Files to cover: `preferences.xml`, `strings.xml`
  - Do not modify, commit files outside of the `-pt` folder.
  - Adapt the date format in `app_hardcoded_subtitle` to the country.
  - Ensure that [`plurals`](https://developer.android.com/guide/topics/resources/string-resource#Plurals) are covered (`schedule_changes_dialog_number_of_sessions`, `schedule_changes_dialog_being`, `notifications_schedule_changes`). The _Translation Editor_ might not point them out.
- [ ] Complete and update translations in `app/src/ccc36c3/res/values-pt` (becomes visible when you change the _Build Variant_)
- [ ] Complete and update translations in `app/src/cccamp2019/res/values-pt` (becomes visible when you change the _Build Variant_)

# Related
- [Localize the UI with Translations Editor](https://developer.android.com/studio/write/translations-editor.html)
- Please read the [contribution guide](https://github.com/EventFahrplan/EventFahrplan/blob/master/CONTRIBUTING.md) before you start working.

---

:left_speech_bubble: If you want to work on this issue please leave a "I like to work on this" comment here.

### PR DESCRIPTION
**Pull request by larissayasin**
# Description
Added missing strings in` app/src/main/res/values-pt` and reviewed portuguese strings.

Resolves #329

### CODE REVIEW

File: app/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -79,7 +83,7 @@
     <string name="libraries">
 		Este aplicativo usa as seguintes bibliotecas:
 		Kotlin, Google Support Library, BetterLinkMovementMethod, Email Intent Builder,
-		Espresso, OkHttp, SnackEngage e TraceDroid
+		Engelsystem, Espresso, OkHttp, SnackEngage e TraceDroid
```
Comment (johnjohndoe) [inline app/src/main/res/values-pt/strings.xml:86]: Nice spotting this :+1:

**Comment by johnjohndoe** [comment]
@romeroclaudino Are you fine with the translation? Anything to add or change?

**Review by romeroclaudino — APPROVED** [review]
Good job @larissayasin! @johnjohndoe, I wouldn't change a thing.

**Review by johnjohndoe — APPROVED** [review]
Thank you very much for the translation update :+1: 

I rebased your branch on top of the latest commit on `master` and fixed an indentation (tabs -> spaces) at `alarms_empty` which was there before.

**Comment by johnjohndoe** [comment]
Happy new year @larissayasin & @romeroclaudino! I started managing translations for the EventFahrplan at [Crowdin](https://crowdin.com/project/eventfahrplan). I like to invite you to join there if you like.

**Pull request merged**